.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Potentially, although the launch.json is generally advised to be added to SCM. Locally, this can be overriden using [workspace settings](https://code.visualstudio.com/docs/editor/workspaces#_workspace-settings).

Reply:
Can you point to where it is advised to check it in, I've personally not worked on a project where I've seen this checked in

Reply:
I myself have not worked with it a lot either, but it's done by home-assistant/core as well, from which I've largely copied this setup from. I liked it there, which is why I added it, but I'm fine with it gone as well.

Reply:
Ok, lets keep it for now and we'll see how we get on